File: 이상한문자만들기/java/20221020/src/SolutionTest.java
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

class SolutionTest {

    @Test
    void process() {
        Solution solution = new Solution();

        assertEquals("TrY HeLlO WoRlD", solution.process("try hello world"));
    }
}

File: 이상한문자만들기/javascript/20221020/src/solution.test.js
import { changeCase, lowerCaseAtOddIndex } from './solution';

test('changeCase', () => {
  expect(changeCase('try hello world')).toBe('TrY HeLlO WoRlD');
});

test('lowerCaseAtOddIndex', () => {
  expect(lowerCaseAtOddIndex('AABBCC')).toBe('AaBbCc');
  expect(lowerCaseAtOddIndex('TRY HELLO WORLD')).toBe('TrY HeLlO WoRlD');
});
